misk/src/main/kotlin/misk/config/ConfigAdminAction.kt

```diff
@@ -14,7 +14,7 @@ class ConfigAdminAction : WebAction {
   @Inject @AppName lateinit var appName: String
   @Inject lateinit var environment: Environment
 
-  @Get("/_admin/api/config/all")
+  @Get("/api/_admin/config/all")
```

Comment:
👍🏻

BTW, the _ in _admin _might_ be unnecessary. We generally use `_` as a URL prefix for internal-facing URLs and don’t send those through our HTTP gateway. But for Misk we aren’t doing that, the `_` is just a naming convention to hint to the reader that it’s internal only.

I’d be tempted to rename this to either `/_api/admin/config/call` (keeping the hint) or perhaps `/api/admin/config/call` (because it’s not a user-facing URL anyway)

Reply:
I guess the initial reasoning for maintaining the `_admin` in the endpoints was to maintain consistency between the url domain given to a tab `_admin/config` and their related endpoints. That didn't come to fully pass given the complexities of matching routes and only passing web ones onto related servers/resources and not losing the endpoint calls within that, thus `/api/_admin/config`. 

1) If we know for sure that there will not be any internal api calls to a future dashboard for `/admin/config/` then I think it'd be okay to go with the suggested `/_api/admin/config`. 

2) Or we abandon the initial goal of shared url namespaces with the related front end view tabs and say that the scoping is module based regardless of whether it goes to a view at `/admin/config` or `/_admin/config`. Then all api endpoints within the config module would live at `/api/config`.

Leaning towards 2, since the shared url domain between view and API seems to be already broken enough that it might make sense to go with a simpler url scheme for endpoints. Thoughts?

Reply:
Yeah I think 2. I’d like the user-facing URLs to have `_` in ’em for social reasons. APIs don’t need ’em.

 * `/_admin/config/` I visit this in my browser
 * `/api/config/` the above URL makes its API calls here